src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/ManifestTest.cs

```diff
@@ -1118,5 +1121,48 @@ public void SupportedOSPlatformVersionErrors (string minSdkVersion, string suppo
 			}
 		}
 
+		[IntentFilter (new [] { "myaction" }, DataPathSuffix = "mysuffix", DataPathAdvancedPattern = "mypattern")]
```

Comment:
I think this test is providing a false sense of security, as the "singular" `DataPathSuffix` + `DataPathAdvancedPattern` attribute *uses the same values* as the "plural" `DataPathSuffixes` and `DataPathAdvancedPatterns` form on the following line.

These should use *different* values, so that we can more easily verify that both are working.

That said, based on the current `expected`, I think [my previous feeling](https://github.com/xamarin/xamarin-android/pull/8261/files#r1296575089) is wrong, because the test method uses `.First()`, which will be the "singular' data, and that's working.  Yay!  (But I don't understand how or why it's working…)

Please update this and the following line so that unique values are used, a'la:

```csharp
[IntentFilter (new [] { "singularAction" },
    DataPathSuffix = "singularSuffix",
    DataPathAdvancedPattern = "singularPattern")]
[IntentFilter (new [] { "pluralAction" },
    DataPathSuffixes = new [] { "pluralSuffix1", "pluralSuffix2" },
    DataPathAdvancedPatterns = new [] { "pluralPattern1", "pluralPattern2" })]
```

Reply:
Good call, tests are much clearer now.